src/js/components/chat-input.jsx

```diff
@@ -38,9 +38,17 @@ export class ChatInputComponent extends Component {
 
     resizeInput(attempt = 0) {
         const node = findDOMNode(this);
-        if (node.offsetWidth - this.refs.button.offsetWidth > 0) {
+
+        const nodeRect = node.getBoundingClientRect();
+        const buttonRect = this.refs.button.getBoundingClientRect();
+
+        // use floor on widget and ceil on button to get worst rounding scenario
```

Comment:
And I prefer to be on the safe side and not trust the browser to round the value to what it needs to be.